src/controllers/commands_controller.js

```diff
@@ -55,12 +55,6 @@ function thingNotFound(response, name) {
   });
 }
 
-function invalidForDevice(response) {
-  response.status(400).json({
-    message: 'Sorry, I\'m afraid I can\'t do that.',
-  });
-}
-
```

Comment:
R-

I suggest instead changing the message to "Sorry Dave, I\'m afraid I can\'t do that." This has the dual benefits of:
1. Being a more accurate cultural reference
2. Freaking out at least two users I know of

I am half serious.

P.S. Hi James!

Reply:
🤔 
Greetings!